lib/scope.js

```diff
@@ -237,6 +237,10 @@ AST_Scope.DEFMETHOD("init_scope_vars", function(nesting){
 AST_Lambda.DEFMETHOD("init_scope_vars", function(){
     AST_Scope.prototype.init_scope_vars.apply(this, arguments);
     this.uses_arguments = false;
+
+    var symbol = new AST_VarDef({ name: "arguments" });
```

Comment:
The back story: https://github.com/avdg/UglifyJS2/commit/5ab518a526d7108aceebc86c6819852b7ad77fe6#commitcomment-15410627